db/sql/150_stop_correspondence_schema.sql

```diff
@@ -1,13 +1,15 @@
--- 
--- Tables for stop correspondence analysis.
--- TODO: Move these tables later to a separate schema,
---       e.g. CREATE SCHEMA stopcorr; CREATE TABLE stopcorr.observation ... etc.
---       This way, objects related to particular business cases
---       are easy to group and identify together on db side.
--- 
+---
+--- Tables for stop correspondence analysis.
+--- TODO: Move all tables under stopcorr schema
+---       This way, objects related to particular business cases
+---       are easy to group and identify together on db side.
+---
+CREATE SCHEMA stopcorr;
+COMMENT ON SCHEMA stopcorr IS
```

Comment:
Created the schema already but didn't want to move existing tables under it yet. Perhaps better to do it in another PR